### PR TITLE
chore: Bring back and update auction banner

### PR DESCRIPTION
**Pull request by Chef-Cheems**

### CODE REVIEW

File: src/config/localization/translations.json [hunk 1]
```diff
@@ -1304,7 +1304,7 @@
   "This page is for projects to bid for farms.": "This page is for projects to bid for farms.",
   "If you’re not a whitelisted project, you won’t be able to participate, but you can still view what’s going on!": "If you’re not a whitelisted project, you won’t be able to participate, but you can still view what’s going on!",
   "through community auctions so far!": "through community auctions so far!",
-  "5 Winners": "5 Winners",
+  "%num% Winners": "%num% Winners",
```
Comment (hachiojidev) [inline src/config/localization/translations.json:1307]: My man!

File: src/views/Home/index.tsx [hunk 2]
```diff
@@ -64,6 +66,13 @@ const Home: React.FC = () => {
             <UserBanner />
           </UserBannerWrapper>
         )}
+        <Flex
+          pt={[account ? '220px' : '0', null, null, account ? '76px' : '0']}
+          mt={[account ? '0' : '-16px', null, null, account ? '0' : '-48px']}
```
Comment (hachiojidev) [inline src/views/Home/index.tsx:71]: I won't even ask.

**Review by hachiojidev — APPROVED** [review]

**Review by ChefKai — APPROVED** [review]

File: src/views/Home/index.tsx (hunk 2 above)
Comment (Chef-Cheems) [reply src/views/Home/index.tsx:71]: This is just revert of what was previously there 😄  I think Hutch did the banner 👀

Comment (ChefHutch) [reply src/views/Home/index.tsx:71]: hahahahha.

Basically the calcs to push everything else on the home page below the absolute-positioned user banner.

**Review by ChefHutch — APPROVED** [review]

**Pull request merged**